src/js/components/filmList/FilmList.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import FilmListElement from './filmListElement/FilmListElement';
+
+import './FilmList.css';
+
+function filmMapCallback(elem) {
+  return <FilmListElement
+      key={elem.id}
+      filmName={elem.title}
+      yearOfIssue={elem.release_date}
+      genre={elem.genres[0]}
```

Comment:
I would pass all genres into the compoenent, or create appropriate data structure in smart compoenent.